src/css/custom.scss

```
@@ -557,18 +559,19 @@ body .DocSearch {
 // External link icon indicator.
-a[target="_blank"]::after {
+.theme-doc-markdown a[target="_blank"]::after {
   content: "";
   display: inline-block;
   width: 0.75em;
-  height: 0.75em;
+  height: 0.609375em;
   margin-left: 0.2em;
   vertical-align: baseline;
+  transform: rotate(-45deg);
   background-color: currentColor;
```

Stylelint (17.13.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)